packages/application-shell/src/components/quick-access/create-commands.js

```diff
@@ -312,16 +310,15 @@ export default ({
               to: `/${project.key}/settings/product-types`,
             },
           },
-          featureToggles.developerSettings &&
-            hasPermission(permissions.ManageProject, project.permissions) && {
-              id: 'go/settings/developer',
-              text: intl.formatMessage(messages.openDeveloperSettings),
-              action: {
-                type: 'go',
-                to: `/${project.key}/settings/developer/api-clients`,
-              },
+          hasPermission(permissions.ManageProject, project.permissions) && {
+            id: 'go/settings/developer',
+            text: intl.formatMessage(messages.openDeveloperSettings),
+            action: {
+              type: 'go',
+              to: `/${project.key}/settings/developer/api-clients`,
             },
-          featureToggles.customApplicationsSettings &&
```

Comment:
This was actually wrong. There is no flag like this and the flag requested is called differently. Hence, quick access for custom app settings was never working to how I see it.